tools/ci_build/github/apple/test_ios_packages.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import pathlib
+import shutil
+import subprocess
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+REPO_DIR = os.path.normpath(os.path.join(SCRIPT_DIR, "..", "..", "..", ".."))
+
+
+def _test_ios_packages(args):
+    # check if CocoaPods is installed
+    if shutil.which('pod') is None:
+        if args.fail_if_cocoapods_missing:
+            raise ValueError('CocoaPods is required for this test')
+        else:
+            print('CocoaPods is not installed, ignore this test')
+
+    # Now we need to create a zip file contains the framework and the podspec file, both of these 2 files
+    # should be under the c_framework_dir
+    c_framework_dir = args.c_framework_dir.resolve()
+    if not c_framework_dir.is_dir():
+        raise FileNotFoundError('c_framework_dir {} is not a folder.'.format(c_framework_dir))
+
+    framework_path = os.path.join(c_framework_dir, 'onnxruntime.framework')
+    if not pathlib.Path(framework_path).exists():
+        raise FileNotFoundError('{} does not have onnxruntime.framework'.format(c_framework_dir))
+
+    # create a temp folder under repo dir
+    import tempfile
+    with tempfile.TemporaryDirectory() as temp_dir:
+        # temp_dir = '/Users/gwang/temp/eee'
+
+        # create a zip file contains the framework
```

Comment:
># create a zip file contains the framework [](http://example.com/codeflow?start=8&length=42)

it might be useful to have a separate function/script which creates the framework archive that we can use to prepare the release package too.

Reply:
Will do this while create the packaging pipeline, add comment